example/snippets.cpp

```diff
@@ -0,0 +1,805 @@
+//
+// Copyright (c) 2022 Alan Freitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CppAlliance/url
+//
+
+//[snippet_headers_1
+#include <boost/url.hpp>
```

Comment:
This file should not be in examples/ because it doesn't show best practices (such as `using boost::urls`). It should be in test/